### ISSUE TITLE
Android application is to slow

### ISSUE THREAD

**Issue by baco** [mattermost/mattermost-mobile#180]
The android application is to slow, sometimes it takes two or three clicks into menu drawers to unveil them and they do not respond to swiping from the borders (the hamburger-menu should).

Compared to Slack's Android application I takes a lot of time to change between channels.

**Comment by lfbrock** [comment]
Hi @baco - are you using the Android application from the Google Play store? 

This repo is for new mobile apps we're working on, which should help with the slowness. They're planned for a beta release in March.

**Comment by baco** [comment]
Yes, sorry, I've noticed that when I stumped into that project after submitting this, I should have closed this issue then.

Now that I am here, I've noticed there are the instructions for building the packages but I am not experienced myself in building with all the android toolchain.  I'd be willing to test any alpha versions that are laying around. By any chance, is there an alpha build of the APK I could try as a tester to report any issues found?
